user_guide/data_management_tags.rst

```diff
@@ -0,0 +1,152 @@
+.. _user-guide-tags:
+
+Tags
+====
+
+Tag is non-hierarchical keyword or phrase assigned to a record. First of all, tags provide additional information about 
+the records to all the system users. Tags can be assigned to any records of any entity and used for  
+:ref:`filtering <user-guide-filters-management>`, they can be used to create a report only for the records that 
+have/don't have a specific tag and generate targeted segments for mailings and marketing campaigns.
+Moreover, you can easily find all the records with a specific tag, even not knowing 
+their other details from anywhere in the system as described hereunder.  
+
+
+Add a Tag to a Record
+---------------------
+
+You can add a tag from the grid or from a View page of a specific record.
+
+Add a Tag from the Grid
+^^^^^^^^^^^^^^^^^^^^^^^
+
+The tag column is by default available on the :ref:`grids <user-guide-ui-components-grids>`. If they are not, you can 
+:ref:`adjust the table settings <user-guide-ui-components-grids-change-table>` and add the column. Adding a tag is not 
+different from :ref:`editing other properties <user-guide-ui-components-grid-edit>` from a grid:
+
+- Double-click the tag cell of the record. 
+
+- Select one or several of the existing tags from the list, and/or enter a new tag in the field.
+
+  |TagsfGrid|
+  
+- Start entering the tag, and only matching tags will be left in the selector.  
+  
+- If you have entered a new tag, it will also appear in the selector. Select it to add. This way you can add one or 
+  several tags that contain one or several words.
+  
+    |TagsfGrid1|
+  
+- Click the check to confirm the action.
+
+.. _user-guide-tags-add:
+
+Add a Tag from the View page
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In order to add a tag to a record from its :ref:`View page <user-guide-ui-components-view-pages>` of a record:
+
+- Go to the *"Tags"* field.
+
+  |TagsfView|
+
+- Click the |IcEdit| icon, and the tag field will appear.
+
+  |TagsfView1|
+  
+- Start entering the tag, and only matching tags will be left in the selector.  
+  
+- If you have entered a new tag, it will also appear in the selector. Select it to add. This way you can add one or 
+  several tags that contain one or several words.
+  
+  |TagsfView2|
+  
+- Click the check to confirm the action.
+
+Manage Tags
+-----------
+
+You can see all the tags available in the system and amount of times they were assigned to a record, in the Tags 
+:ref:`grid <user-guide-ui-components-grids>`.
```

Comment:
Edit: You can see all the tags available in the system, as well as the amount of times they were assigned to a record, in the Tags :ref:`grid <user-guide-ui-components-grids>`.